tests/terraform/terraform-tests.success.txt

```diff
@@ -0,0 +1,4 @@
+route53:
```

Comment:
nit: Imho, since it's interpreted as a yaml file, the filename extension should indicate that (i.e. the filename should be `terraform-tests.success.yaml` or something similar). I think the name is a bit misleading, since "success" somewhat implies that it's the result of a previous run (which might be the reason for the name in the first place?).
Maybe allowlist would be better? Or just `terraform-tests.yaml` with a comment on top?

Reply:
renamed file